### DISCUSSION TITLE
Need to know the impact of Dotnet Boxed Graphql templates with .NET Core 3.0

### DISCUSSION THREAD

**Discussion by santoshpatro** [Dotnet-Boxed/Templates#427] [Q&A]
Hi Rehaan,

I have installed Dotnet-Boxed Templates for GraphQL and using ASP.NET Core 3.0 to develop GraphQL APIs. I need your inputs for the below mentioned queries:

1. Program.cs page content is completely different (GraphQL NET Boxed template) compared to the vanilla ASP.NET Core 3.0 project.
2.  Want to know will be there any templates aligned to NET Core 3.0 in the near future.

Any help on this is much appreciated.

Thanks,
Santosh

**Comment by VictorioBerra** [comment]
For 1 this is not a standard aspnet core template pack, this is not a Microsoft provided template. These templates have more features and customization than the default templates.

For 2 see #411 TLDR: were working on it.

**Comment by santoshpatro** [comment]
@VictorioBerra  Thanks for your response :)

**Comment by santoshpatro** [comment]
@VictorioBerra In the Startup.cs , method:  CreateWebHostBuilder() 

// Used for IIS and IIS Express for in-process hosting. Use UseIISIntegration for out-of-process hosting.
.UseIIS()

In case I want to host in Azure Kubernetes Cluster , what should I use ?

Thanks,
Santosh

**Comment by VictorioBerra** [comment]
I do not use AKS but I'd imagine you still need a reverse proxy, you obviously wouldn't use IIS. Probably just use kestrel and no HTTPS since your reverse proxy will do SSL termination

**Comment by santoshpatro** [comment]
@VictorioBerra  Thanks a lot for your response :)

**Comment by RehanSaeed** [comment]
Thanks @VictorioBerra! I added you as a contributor if you don't mind, so you should be able to manage issues. You should get an invitation if you are interested in accepting.

**Comment by VictorioBerra** [comment]
Sounds great, will accept. Happy to help!
